scipy/optimize/_minpack_py.py

```diff
@@ -864,6 +869,22 @@ def curve_fit(f, xdata, ydata, p0=None, sigma=None, absolute_sigma=False,
     suggesting that the optimal values of these parameters are ambiguous and
     that only one of these parameters is needed in the model.
 
+    Fit errors can also be caused by scale differences between parameters.
+    An orders of magnitude difference between function parameters can give
+    inaccurate results or entirely fail to find a result.
+
```

Comment:
If the optimal parameters of `f` differ by more than four orders of magnitude, the resulting fit can be inaccurate. Sometimes, `curve_fit()` can fail to find any results.